tapeout_16/scripts/layout_Tile.tcl

```diff
@@ -72,7 +74,16 @@ createRouteBlk -name cut01M1 -layer M1 -cutLayer all -box [list 0 [expr $height
 createRouteBlk -name cut02M1 -layer M1 -cutLayer all -box [list 0 -1 $width 0.5]
 
 if $::env(PWR_AWARE) {
-   modifyPowerDomainAttr AON -box 49.95 [expr $height - 9.792] 64.98 [expr $height - 1.152]  -minGaps 0.576 0.576 0.18 0.18
+   set pp_x 0.09
+   set pp_y 0.576
```

Comment:
Could we move these numbers out into params.tcl and name them things that explain what they are?

Reply:
I can change pp_x to polypitch_x if that helps. Rest all is self explanatory.

Reply:
pp_y is not self-explanatory. I'd also like to have all technology related parameters in params.tcl for easier technology-porting in the future.

Reply:
wait I guess it's also the poly_pitch... I thought it was something else. Still, I'd like the technology related parameters as centralized as possible.

Reply:
yup, that's polypitch too, one for X and Y. I have moved all parameters to the params.tcl.

Reply:
ok once you push that change I think this can be merged in